### DISCUSSION TITLE
PLOT stuck at 0%

### DISCUSSION THREAD

**Discussion by grafixstudio** [Chia-Network/chia-blockchain#3559] [Q&A]
My plotting status says 0% and does not move. I notice files being created in the temporary folder. I don't know what to think. 

I had to restart the application once and it was still bugging, sometimes it wants me to add a plot again but when I do that it says I already have a plot selected.

https://ibb.co/VYGhC6Z
https://ibb.co/CPMn41R

**Comment by huiui** [comment]
How long it was stuck? It may take up to 45mins to pass this stage from my experience.  Some time its really fast, sometime takes longer, not sure why is that.

**Reply by grafixstudio** [reply]
I was stuck forever...left it on all night and still just said" Loading"

**Comment by moshenskyDV** [comment]
Could you please share plotting log? (three dots on the right side of plot)

**Reply by grafixstudio** [reply]
Plotting just says "Loading"  and nothing happens with it. I had it on the entire night.

<a href="https://ibb.co/HGWGHxz"><img src="https://i.ibb.co/HGWGHxz/plotstuck.png" alt="plotstuck" border="0"></a>

**Comment by grafixstudio** [comment]
Hi, 

Thanks for the quick reply

The first time I started everything, the log file had some data but after I had to reboot,  it just says Loading

I have an external hard drive for the plot and my computers SSD for the temp plot.

I just tried to have it on all night. I just took the screenshot.






» Har du sett min senaste video på YouTube?

Med vänlig hälsning
Antonio Riveras 
Phone:  00 (46) 707 52 11 64 
E-mail:   ***@***.***  
Website: https://grafixstudio.se
Blog: https://antonioriveras.se  



På 3 maj 2021 kl. 23:30:55, Moshenskyi Danylo ***@***.***) skrev:

Could you please share plotting log? (three dots on the right side of plot)

**Reply by grafixstudio** [reply]
<a href="https://ibb.co/HGWGHxz"><img src="https://i.ibb.co/HGWGHxz/plotstuck.png" alt="plotstuck" border="0"></a>
